tabbycat/draw/generator/powerpair.py

```diff
@@ -172,7 +172,7 @@ def _pullup(self, brackets, pos):
         Operates in-place. Does not remove empty brackets."""
         pullup_needed_for = None
 
-        for points, teams in brackets.items():
+        for _, teams in brackets.items():
```

Comment:
If we only need the values (not the keys), how about `.values()`?

Reply:
Yes, that should make it easier.